src/proxy/lib/forward.js

```diff
@@ -0,0 +1,26 @@
+const { dummyServer } = require('../../config');
+const superagent = require('superagent');
+
+async function forwardRequest(req, res) {
```

Comment:
There's libraries you can use to create proxies. For example as you're using express https://github.com/chimurai/http-proxy-middleware